lib/reverse_sentence.rb

```diff
@@ -1,6 +1,45 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n), where n is the number of characters in the string.
```

Comment:
This is a good start but there is room for improvement. For example, this fn calls two other functions. How do the time complexities of the other functions come together to be simplified to O(n)?